docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: "3"
+services:
+  stecker-club-monitor:
+#    depends_on:
+#      - stecker-cdn
+    build:
+      context: .
+      dockerfile: ./Dockerfile
+    image: "club-monitor"
+    hostname: club-monitor
+    links:
+      - club-monitor-db:mysql
+    ports:
+      - "8082:80"
+    volumes:
+      - /var/lib/steckercontent/storage/club-monitor/uploads:/var/www/html/common/uploads/
+      - /var/lib/steckercontent/storage/club-monitor/config.php:/var/www/html/common/config.php
+    labels:
+      traefik.enable: "true"
+      traefik.frontend.rule: "PathPrefixStrip:/club-monitor"
+    networks:
+      - stecker
+  club-monitor-db:
+    image: mysql:5.7
+    volumes:
+       - /var/lib/steckercontent/mysql/club-monitor:/var/lib/mysql
+    restart: always
+    env_file:
+       - sqlpwd.env
```

Comment:
Machst Du dazu bitte noch einen Eintrag in die gitignore und ein Example daneben?